pkg/operator/controller/ingress/load_balancer_service_test.go

```diff
@@ -11,15 +13,78 @@ import (
 
 func TestDesiredLoadBalancerService(t *testing.T) {
 	testCases := []struct {
+		description  string
 		strategyType operatorv1.EndpointPublishingStrategyType
+		lbStrategy   operatorv1.LoadBalancerStrategy
+		proxyNeeded  bool
 		expect       bool
 	}{
 		{
+			description:  "external classic load balancer with scope",
 			strategyType: operatorv1.LoadBalancerServiceStrategyType,
+			lbStrategy: operatorv1.LoadBalancerStrategy{
+				Scope: operatorv1.ExternalLoadBalancer,
+			},
+			proxyNeeded: true,
+			expect:      true,
+		},
+		{
+			description:  "external classic load balancer without scope",
+			strategyType: operatorv1.LoadBalancerServiceStrategyType,
+			lbStrategy: operatorv1.LoadBalancerStrategy{
+				Scope: operatorv1.ExternalLoadBalancer,
```

Comment:
This test case is "without scope", so it seems like this line should be omitted.